docs/recipes/reducers/10-InitializingState.md

```diff
@@ -0,0 +1,118 @@
+# Initializing State
```

Comment:
Worth looking at.  I definitely wrote the `combineReducers` section well before I pulled over Dan's explanation from SO.  Could certainly cross-link from "`combineReducers`" to "Initializing State"